client/my-sites/patterns/components/pattern-preview/style.scss

```diff
@@ -104,6 +104,11 @@
 	}
 }
 
+.pattern-preview--page {
+	break-inside: avoid;
+	margin-bottom: 16px;
```

Comment:
Why did you decided to add margin-bottom instead of increasing `gap` for `.pattern-gallery`? In this case we would have 1 place instead of 2

Reply:
`row-gap` doesn't have any effect with CSS columns, unfortunately